.circleci/config.yml

```
@@ -75,7 +75,13 @@ jobs:
     steps:
       - checkout
       - <<: *restore_dependency_cache_unix
-      - run: npm run test
+      - run: npm run build
+      - run: npm run test:unit -- --browsers Chrome,Firefox
+      - run:
+          command: npm run start
+          background: true
+      - run: npm run test:integration -- browser=chrome
+      - run: npm run test:integration -- browser=firefox
```

Can we change this so that it's consistent with test:unit?

```suggestion
      - run: npm run test:integration -- --browsers Chrome,Firefox
```